Framework/AnalysisSupport/src/AODJAlienReaderHelpers.cxx

```diff
@@ -107,6 +107,8 @@ std::vector<std::string> getColumnNames(header::DataHeader dh)
       return columnNamesTrait(typename StoredTracksCovMetadata::table_t::persistent_columns_t{});
     } else if (description == "TRACK:EXTRA") {
       return columnNamesTrait(typename TracksExtraMetadata::table_t::persistent_columns_t{});
+    } else if (description == "HMPID") {
+      return columnNamesTrait(typename TracksHMPID::table_t::persistent_columns_t{});
```

Comment:
This is not needed (it is only needed for the tracks table)

Reply:
I moved the HMPID table and renamed `HMPIDs`